packages/dashboard/src/components/template/you-might-like/you-might-like.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import YouMightLikeCard from './you-might-like-card';
+
+const trader_types = [
+    {
+        code: '1',
+        bg: 'trade-type/bg-digital-option.png',
+        title: 'Digital options',
+        sub_title: localize('Earn fixed payouts by predicting an asset price movements'),
+    },
+    {
+        code: '2',
+        bg: 'trade-type/bg-multipliers.png',
+        title: 'Multipliers',
+        sub_title: localize('Amplify your gains without losing more than your stake.'),
+    },
+    {
+        code: '3',
+        bg: 'trade-type/bg-margin.png',
```

Comment:
we need to prefix this with `{config_store.asset_path}/images`

Reply:
This config is being given in `you-might-like-card.tsx` file. It is just the last part of the url which is changing.